### PR TITLE
Load asks/bids data via rpc if db result is outdated

### PR DESCRIPTION
**Pull request by 0xcrust**
## Description
Fallback to rpc for loading orderbook asks and bids if results retrieved from database were last updated > 60 seconds ago. 

[Clickup task](https://app.clickup.com/t/86cv39gjj)

## Types of changes

- [ ] Technical Debt (non-breaking change which removes unused code/assets)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] The related ClickUp task has been linked to this PR
- [x] The person creating the pull request is listed in "Assignees"
- [ ] Change requires updated documentation

### CODE REVIEW

File: src/context/orderbook.tsx [hunk 1]
```diff
@@ -133,8 +135,25 @@ export const OrderBookProvider: FC<{ children: ReactNode }> = ({ children }) =>
...
+    if (difference > 60 * MILLISECONDS_IN_SECOND) {
+      const { connection } = useConnectionConfig()
+      let [asks, bids] = await Promise.all([
+        loadSlab(connection, activeProduct.asks),
+        loadSlab(connection, activeProduct.bids)
+      ])
+      orderbookBids = [...bids.items(false)].map((bid) => [bid.leafNode.getPrice(), bid.leafNode.baseQuantity])
+      orderbookAsks = [...asks.items(false)].map((ask) => [ask.leafNode.getPrice(), ask.leafNode.baseQuantity])
+    } else {
+      orderbookBids = res.data?.bids.map((item) => [item.price, item.size])
```
Comment (AlexOfc) [inline src/context/orderbook.tsx]: Here you do two checks if data is not nullable.

File: src/context/orderbook.tsx [hunk 2]
```diff
@@ -133,8 +135,25 @@ export const OrderBookProvider: FC<{ children: ReactNode }> = ({ children }) =>
       pairName: activeProduct.pairName,
       devnet: isDevnet
     })
-    const orderbookBids = res.data?.bids.map((item) => [item.price, item.size])
-    const orderbookAsks = res.data?.asks.map((item) => [item.price, item.size])
+    const lastUpdated = new Date(res.data?.updatedTime).getTime()
```
Comment (AlexOfc) [inline src/context/orderbook.tsx]: All the code below that work with timestamp doesn't make sense if data is nullable.

Comment (usgoose) [reply src/context/orderbook.tsx]: Thanks ser taking a look! Feel free to ping me on telegram btw @usgoose happy to chat

**Review by candidatek — APPROVED** [review]

**Comment by 0xcrust** [comment]
Looks good to be merged @0xabstracted!

**Pull request merged**